### DISCUSSION TITLE
Automatic pump speed detection for CPU

### DISCUSSION THREAD

**Discussion by AngelOfMors** [Rem0o/FanControl.Releases#2781] [General]
I have an uncontrolled pump, it works at maximum speed. When the speed is determined, then at low voltage values, the pump turns off, which leads to heating of the CPU. It is necessary to add an option when the automatic detection of fans occurs, that the pump is uncontrolled.

**Comment by Rem0o** [comment]
Is the heating significant enough? The stop time should be relatively short as it starts ramping the % back as soon as stop is detected.

**Comment by AngelOfMors** [comment]
It was noticeable, I have i7-14700K

**Reply by Rem0o** [reply]
I expect noticable, but I want to know if its significant.  Like does it pegs it to 100c for 10 seconds or does it increase to like 70-80 for 5 seconds?

**Reply by AngelOfMors** [reply]
The second option is closer.
